### PR TITLE
Adds data/jobs application-level query functions

### PR DESCRIPTION
**Pull request by hoddr**

### CODE REVIEW

File: voxel51/apps/api.py [hunk 1]
```diff
@@ -228,6 +228,52 @@ def delete_analytic(self, analytic_id):
         res = self._requests.delete(endpoint, headers=self._header)
         _validate_response(res)
 
+    # DATA ####################################################################
+
+    def query_application_data(self, data_query):
+        '''Performs a customized data query at the application level (e.g.
```
Comment (lethosor) [inline voxel51/apps/api.py]: This should be just one line per https://github.com/google/styleguide/blob/gh-pages/pyguide.md#381-docstrings so I'd move the explanation to a new paragraph.

**Review by lethosor — APPROVED** [review]

File: voxel51/apps/api.py (hunk 1 above)
Comment (brimoor) [reply voxel51/apps/api.py]: @lethosor thanks for reading the style guide closely!!!!

File: voxel51/apps/api.py [hunk 2]
```diff
@@ -228,6 +228,52 @@ def delete_analytic(self, analytic_id):
...
+            includes all application users sans application admins).
```
Comment (brimoor) [inline voxel51/apps/api.py]: Please stop making a distinction between `application users` and `application admins` in documentation. App admins are administrators of the application and cannot upload data, run jobs, and do other user-y things.

The fact that our implementation once had a confusion between these things is not relevant to users of our application.

An example docstring might be:

```py
'''Performs a customized data query at the application level.

Queries using this route return results for all users of the application.
'''
```

**Review by brimoor — COMMENTED** [review]
Commenting rather than requesting changes to avoid blockage, but please do address my comment before merging

**Pull request merged**